scripts/update_blog_ads_links.js

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
```

Comment:
This whole file looks very much like the other script. What are the differences? Can we not organize it into one script?